docs/CHANGELOG.md

```diff
@@ -109,7 +109,7 @@ Entries inside each section should be ordered by type:
 * [Added] Declared compatibility with `jsonschema==4.*`.
 * [Added] `--host` and `--port` parameters for `quilt3 catalog`.
 * [Added] `--no-browser` parameter for `quilt3 catalog`.
-* [Changed] `quilt3 catalog` now requires `quilt3` to be installed with `catalog` extra dependency (`pip install quilt3[catalog]`).
+* [Changed] `quilt3 catalog` now requires `quilt3` to be installed with `catalog` extra dependency (`pip install 'quilt3[catalog]'`).
```

Comment:
No changelog for docs changes, right?

Reply:
Description is also incorrect. We just need to single quote any instances of the extras installs.